### DISCUSSION TITLE
How to get active feedback with respect to pull-request, issues and proposals?

### DISCUSSION THREAD

**Discussion by akoserwal** [go-kratos/kratos#3383] [Q&A]
Hi Team,

A few of the challenges I see with the Kratos projects: 

- How do we tag maintainers on the pull request? Tag active reviewers?
- There no active discord channel to discuss issues and get quick inputs?
- The open issues discussion has been stalled for far too long.

Example
Open Pr for a month, and I didn't receive any active feedback.
https://github.com/go-kratos/kratos/pull/3359

**Comment by Windfarer** [comment]
I apologize for the delay in addressing this Pull Request, we will process it as soon as possible.

**Reply by akoserwal** [reply]
No problem, I would be more interested in improving overall process/workflow to avoid such challenges in future

Some of the options:

* How do we tag maintainers on the pull request? Tag active reviewers?
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews
Help in auto-assigning or allow assigning reviewers to the pull request

* The open issues discussion has been stalled for far too long.
https://github.com/actions/stale

* There no active discord channel to discuss issues and get quick inputs?
go-kratos channel on Discord isn't active or maintained

Please see how such challenges can be improved
